lib/engine/game/g_18_esp/step/acquire.rb

```diff
@@ -90,10 +90,14 @@ def choices
           end
 
           def can_swap?
```

Comment:
An alternative implementation could be to move line 95-97 plus the unless into a method that returns true/false.
And then
return true if is_some_method

followed by merged_token_in_shared_city? content.
Not sure which is most readable?

Reply:
i think this is readable as it is, i don't think refactoring will make it more readable